client/my-sites/checkout/checkout-thank-you/difm/style.scss

```diff
@@ -0,0 +1,96 @@
+.difm-lite-thank-you {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	@include breakpoint-deprecated( '<660px' ) {
```

Comment:
Instead of `breakpoint-deprecated`, we can use the `break-*` breakpoints from Gutenberg. Reference here: https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#media-queries

Reply:
Fixed!